templates/repo/migrate/migrating.tmpl

```diff
@@ -36,10 +36,11 @@
 								<div class="divider"></div>
 								<div class="item">
 									{{if .Failed}}
-									<button class="ui basic red show-modal button" data-modal="#delete-repo-modal">{{.locale.Tr "repo.settings.delete"}}</button>
+										<button class="ui basic red show-modal button" data-modal="#delete-repo-modal">{{.locale.Tr "repo.settings.delete"}}</button>
 									{{else}}
-									<button class="ui basic red show-modal button" data-modal="#cancel-repo-modal">{{.locale.Tr "cancel"}}</button>
+										<button class="ui basic red show-modal button" data-modal="#cancel-repo-modal">{{.locale.Tr "cancel"}}</button>
 									{{end}}
+									<button id="repo_migrating_retry" data-migrating-task-retry-url="{{.Link}}/settings/migrate/retry" class="ui basic button gt-hidden">{{.locale.Tr "retry"}}</button>
```

Comment:
Wait, isn't this button in the wrong place at the moment?
Shouldn't it be inside the `if` block?
If I see this correctly, you also display it next to the `cancel` button which doesn't make much sense to me.